### ISSUE TITLE
Crash on Character Select/Creation

### ISSUE THREAD

**Issue by 18dietzlerjo** [tsunamayo/Starship-EVO#2862]
### Short Description of the issue:
Just downloaded the game, never played before, set up a new character, hit play and watch the loading screen for a few seconds and after about a minute the game completely closes without an error box. Occasionally the unity error manager (I assume) opens, and a loading bar completes and then the game closes.

### Steps to reproduce the issue:
Create a character? it is a fresh install so I am unsure how to replicate.

### Environment
- Starship EVO version: 20w36a
- Graphics Card: Intel UHD 620

### Additional information:
I have an intel i7 with 16 Gb ram and an integrated graphics card. I have never had an issue processing a game though. I initially tried downscaling the resolution but it ended in the same result.
_(Screenshots, Videos, Error logs, Related issues)_

[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5157530/output_log.txt)

**Comment by tsunamayo** [comment]
Yes it seems that this GPU has some issue for now, it is below minimum spec. I will try to make it work if I can get my hand on a PC with such a card. Thanks for your understanding

**Comment by tsunamayo** [comment]
By the way what is your laptop model? Thanks

**Comment by 18dietzlerjo** [comment]
Yeah, I bought this computer thinking it came with a freaking graphics card lol. It's the stock graphics card on the HP spectre x360 13t. I can usually change how much GPU is assigned because it's integrated. I believe it uses some CPU as GPU, so I can try dabbling in that a bit to see if that helps any.
